index.html

```diff
@@ -0,0 +1,41 @@
+</<!DOCTYPE html>
```

Comment:
On front end changes a screenshot of your changes should come along with the PR (put it on the description).